x.consolidation/consolidation.md

```diff
@@ -0,0 +1,517 @@
+# UTXO Consolidation
+
+Unspent Transaction Output (UTXO) consolidation is the act of taking the
+bitcoin value stored in several UTXOs and *consolidating* it into one
+UTXO.  This lowers the number of UTXOs you control while keeping your
+balance roughly the same, increasing the chance that you'll be able to
+create smaller and more affordable transactions at a later time when
+you need to pay a higher feerate in order to obtain a fast confirmation.
+
+## A consolidation example
+
+Imagine a transaction as a rectangle whose height is its feerate, whose
```

Comment:
This is noted in a comment in the gnuplot file, but I deliberately avoided choosing a particular feerate here (and in the other plots, except the last one based on real data) because any feerate we use will probably not match the current feerates at the time people are reading this.  (I think we had this same problem in one of the other chapters, so I wonder if maybe it'd be an idea to invent bogocoins or bogobits or something, describe them in the introduction, and then use them throughout the book.)

Reply:
I agree things like feerate will not age well.

I dont have a strong preference on this so marking resolved.